src/app/modules/entry-live/views/live-bandwidth/live-bandwidth.widget.ts

```diff
@@ -38,11 +38,13 @@ export class LiveBandwidthWidget extends WidgetBase<LiveQoSData> {
   }
 
   protected _onRestart(): void {
-    this._pollsFactory = new LiveBandwidthRequestFactory(this._activationArgs.entryId);
+    this._pollsFactory = new LiveBandwidthRequestFactory(this._activationArgs.entryId, this._activationArgs.countryIn, this._activationArgs.regionIn, this._activationArgs.citiesIn,
```

Comment:
@viktor-kyryltsev-kaltura - same comment regarding passing an object instead of multiple parameters. In line 47 as well

Reply:
done